### PR TITLE
boot: tweak boot chains to support a list of kernel command lines, keep track of model and kernel boot file

### PR DESCRIPTION
**Pull request by bboozzoo**
Some prerequisites for resealing work.

### CODE REVIEW

File: boot/export_test.go [hunk 1]
```diff
@@ -111,3 +113,19 @@ var (
 	ToPredictableBootChains             = toPredictableBootChains
 	PredictableBootChainsEqualForReseal = predictableBootChainsEqualForReseal
 )
+
+func (b *bootChain) SetModel(model *asserts.Model) {
```
Comment (pedronis) [inline boot/export_test.go:117]: SetModelAssertion

File: boot/export_test.go [hunk 2]
```diff
@@ -111,3 +113,19 @@ var (
...
+	b.model = model
+}
+
+func (b *bootChain) GetModel() *asserts.Model {
```
Comment (pedronis) [inline boot/export_test.go:121]: ModelAssertion

**Review by pedronis — APPROVED** [review]
thank you

**Review by cmatsuoka — APPROVED** [review]
Thanks Maciek, lgtm.

**Pull request merged**